src/pages/components/CertificateForm.js

```diff
@@ -37,8 +37,11 @@ const callBodyCrafter = (enVal, sdVal, stVal, edVal, etVal, szVal) => {
   return body
 }
 
-export default function CertificateForm(props) {
-  const { current, update } = useContext(Context)
+export default function CertificateForm() {
+  const { current, update, ...rest } = useContext(Context)
+  const { fetch } = useAxios(false)
+  const navigate = useNavigate()
+
```

Comment:
Another thing, since the page is redirecting to /certificate/123 rather fast and you can't see the design of the green timeline .. maybe a quick and dirty compromise is to temporarly replace

navigate(`/certificate/${originalTokenId}`)

with  


setTimeout(() => {
  navigate(`/certificate/${originalTokenId}`)
}, 60 * 1000)

Reply:
pushed

Reply:
this has been removed as not part of a scope